discover/openapi.go

```diff
@@ -0,0 +1,38 @@
+package discover
```

Comment:
any reason this was placed here vs. within apispecs.go? given how closely related these are?

Reply:
This was result of modeling after existing implementations in IAM.
After trimming all the excess code, it does make sense to move this over.

Moved into `apispecs.go`